source/agora/node/Ledger.d

```diff
@@ -136,7 +145,9 @@ public class Ledger
     public bool onTXSetExternalized (Set!Transaction txs) nothrow @trusted
     {
         scope (failure) assert(0);
-        auto block = makeNewBlock(this.last_block, txs.byKey());
+        Enrollment[] enrollments;
+        auto block = makeNewBlock(this.last_block, txs.byKey(),
+            this.enroll_man.getUnregisteredEnrollments(enrollments));
```

Comment:
`getUnregisteredEnrollments` takes a buffer, but you are passing it an empty array. If you want to reuse memory, you should mark `Enrollment[] enrollments;` as `static`.

Reply:
applied the suggestion.

Reply:
i'm add static.